src/olympia/devhub/forms.py

```diff
@@ -788,39 +787,14 @@ class VersionForm(WithSourceMixin, forms.ModelForm):
     approval_notes = forms.CharField(
         widget=TranslationTextarea(attrs={'rows': 4}), required=False)
     source = forms.FileField(required=False, widget=SourceFileInput)
-    clear_pending_info_request = forms.BooleanField(required=False)
 
     class Meta:
         model = Version
-        fields = ('release_notes', 'clear_pending_info_request',
-                  'approval_notes', 'source',)
+        fields = ('release_notes', 'approval_notes', 'source',)
 
     def __init__(self, *args, **kwargs):
         self.request = kwargs.pop('request')
```

Comment:
do we still need to pass and store the request on the form now?  (it was just used for the `log_and_notify` in save, right?)

Reply:
I originally removed it, but then changed my mind because it meant looking at all the callers, but there seems to be only one view that uses it, so yeah I'll go ahead and remove it.

Reply:
319cee5deb59538bdc1097f40f855cc2b85589db